tutorials/28_Structured_Output_With_OpenAI.ipynb

```diff
@@ -0,0 +1,591 @@
+{
```

Comment:
<blockquote>but passing it in the&nbsp;<code>run</code>&nbsp;method is recommended for better compatibility.</blockquote><p>I don't understand this part.  What do you mean?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-tutorials/pull/412/discussion/#comment-2581346703'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>